lib/internal/errors.js

```diff
@@ -610,6 +610,17 @@ function addNumericalSeparator(val) {
   return `${val.slice(0, i)}${res}`;
 }
 
+// Append `causedByError` to the `error`.
+function addCausedByError(error, message, causedByError) {
+  let msg = message;
+  if (causedByError) {
+    error.cause = causedByError;
+    if (typeof causedByError.message === 'string')
+      msg += ` (caused by: ${causedByError.message})`;
```

Comment:
I don't have strong feelings about this, but I looked through existing errors, and it looks like a more consistent form might be: `+= "because: " + causedByError.message`, in other words, drop the `(...)`. What do you think?

Reply:
@sam-github `ERR_HTTP2_STREAM_CANCEL` was the only similar `Error`(one error with another) that I could find so that I followed its form. I think we would better not to change the message of `ERR_HTTP2_STREAM_CANCEL`.

But I don't have strong feelings too. Either looks good as they describe the reason clearly.